Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM docker.io/centos:centos7
 
 RUN yum install -y python-requests && \
+    curl -sS https://downloads.mariadb.com/MariaDB/mariadb_repo_setup | bash && \
```

Comment:
Do you have a link to the bug? El8 has 10.3.11, want to make sure whatever bug we're hitting is fixed in the EL8 version so we're ready when the time comes.  There's no EL8 packages provided by this script:


```
  supported="The MariaDB Repository supports these Linux OSs, on x86-64 only:
      * RHEL/CentOS 6 & 7
      * Ubuntu 14.04 LTS (trusty), 16.04 LTS (xenial), & 18.04 LTS (bionic)
      * Debian 8 (jessie), 9 (stretch), & 10 (buster)
      * SLES 12 & 15"
```

Reply:
Unfortunately I don't :/ we did not find the actual bug

Reply:
you can find out very easily if it has the bug or not by running the docker container in Minikube 1.5.0. if it starts, the bug was fixed